src/img/container-move.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="30px" height="30px" viewBox="0 0 30 30" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><script xmlns=""/>
+    <!-- Generator: Sketch 41 (35326) - http://www.bohemiancoding.com/sketch -->

```

Comment:
Please remove this comment. Where was this image from also? Is there any copyrights on it?

Reply:
Removed